### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by wslack**
Changed page title and USDS playbook edit

Flagging for @brethauer

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ---
 permalink: /
-title: 18F Partnership Playbook
+title: 18F Delivery: Partnership Playbook
```
Comment (mbland) [inline index.md:3]: This needs to match exactly the `navigation:` entry in `_config.yml`. Running `./go update_nav` should do the trick.

**Comment by wslack** [comment]
How about this, @mbland?

**Comment by brethauer** [comment]
Looks good to me!

File: _config.yml [hunk 2]
```diff
@@ -26,7 +26,7 @@ logoalt: 18F logo
 # Navigation
 # List links that should appear in the site sidebar here
 navigation:
-- text: The 18F Partnership Playbook
+- text: 18F Delivery: Partnership Playbook
```
Comment (mbland) [inline _config.yml:29]: When you've got a string that contains a `:`, you need to wrap it in quotation marks.

File: _config.yml [hunk 3]
```diff
@@ -57,7 +57,7 @@ navigation:
 
 repos:
 - name: Partnership Playbook
-  description: Partnership Playbook
+  description: 18F Delivery: Partnership Playbook
```
Comment (mbland) [inline _config.yml:60]: Same here regarding quotes when the string contains a `:`.

File: index.md (hunk 1 above)
Comment (mbland) [inline index.md:3]: Oh, and we need the quotation marks here, too. :-)

**Pull request merged**